scripts/pi-hole/php/queryads.php

```diff
@@ -23,7 +23,8 @@ function echoEvent($datatext) {
 ini_set("pcre.recursion_limit", 1500);
 function is_valid_domain_name($domain_name)
 {
-    return (preg_match("/^((-|_)*[a-z\d]((-|_)*[a-z\d])*(-|_)*)(\.(-|_)*([a-z\d]((-|_)*[a-z\d])*))*$/i", $domain_name) // Valid chars check
+    return (preg_match("/^-/", $domain_name) !== 1 // Don't allow domains to appear as command line options
```

Comment:
Woudln't it be much more lightweight to do something like `if($domain_name[0] != '-' && ...)`?